### PR TITLE
[Shell] fix for avoid UI hanging on large data from `child_process`

### LINKED ISSUE #13015: [Shell] Raycast freezes if run `top` command in shell extension.
### Extension

https://www.raycast.com/asubbotin/shell

### Raycast Version

1.76.1

### macOS Version

_No response_

### Description

image

### Steps To Reproduce

1. Install Shell extension
2. Execute `top` command in Shell extension
3. freeze!

### Current Behaviour

_No response_

### Expected Behaviour

_No response_

### PR DESCRIPTION
**Pull request by kevinm6**
- fixes #13015

## Description

Avoid use of `string interpolation` (and concat) with **data** from `child_process`.

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/shell/package-lock.json
[the remark this answers is not in the document]
Comment (mil3na) [reply extensions/shell/package-lock.json:1]: Can you please remove it, then?

[the remark this answers is not in the document]
Comment (mil3na) [reply extensions/shell/package-lock.json:1]: You don't remove the file, just the changes made to it :)

[the remark this answers is not in the document]
Comment (kevinm6) [reply extensions/shell/package-lock.json:1]: Reverted to previous version

**Review by mil3na — APPROVED** [review]
LGTM!

**Pull request merged**